demo/component/BarChart.js

```diff
@@ -239,6 +239,10 @@ export default class Demo extends Component {
     console.log(`Pv Bar (${index}) Click: `, data);
   };
 
+  handleDoubleClick = () => {
+    console.log('Double click invoked!');
```

Comment:
Could you also log the provided arguments here, just so the user can see them being passed through to the handler?

Reply:
Yep, not sure why I didn't do that in the first place. Definitely worth showing that.